src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -161,14 +163,97 @@ public void cannotUpdateCirculationRulesWithLostItemPolicyId() {
       overdueFinePoliciesFixture.facultyStandard().getId().toString(),
       UUID.randomUUID().toString());
 
-    Response response = circulationRulesFixture
-      .attemptUpdateCirculationRules(rule, "i");
+    Response response = circulationRulesFixture.attemptUpdateCirculationRules(rule);
 
     assertThat(response.getStatusCode(), is(422));
     assertThat(response.getJson().getString("message"),
       is("The policy i does not exist"));
   }
 
+  @Test
+  public void canUpdateCirculationRulesWithTwentyExistingLoanPolicies() {
+
+    Set<UUID> loanPolicyIds = getSetOfPolicyIds(20);
+    addPoliciesToLoanPolicyFixture(loanPolicyIds);
+
+    loanPolicyIds.forEach(loanPolicyId -> {
+      String rule = circulationRulesFixture.soleFallbackPolicyRule(
+        loanPolicyId.toString(), requestPoliciesFixture.allowAllRequestPolicy().getId().toString(),
+        noticePoliciesFixture.activeNotice().getId().toString(),
+        overdueFinePoliciesFixture.facultyStandard().getId().toString(),
+        lostItemFeePoliciesFixture.facultyStandard().getId().toString());
+      circulationRulesFixture.updateCirculationRules(rule);
```

Comment:
Sonar complains that this is a test without assert.
Can you rename the two updateCirculationRules methods in CirculationRulesFixture to assertUpdateCirculationRules? The assert that the update succeeds with a 204 status.

Reply:
We already have the method attemptUpdateCirculationRules(rule) that returns Response. 
Updated and added assert.